tests/base/parser/bad/gold/type_t-map-cid-cid-lparen-cid-underscore.scilla.gold

```diff
@@ -1,13 +1,11 @@
 {
   "errors": [
     {
-      "error_message":
-        "This type annotation is not closed properly, or is missing parentheses to group the types, or has a missing or misplaced '='.\n",
+      "error_message": "This is an invalid map type, the map value type is likely incorrect. The map type must be a storable type, i.e., a base type, a valid ADT, or a map type.\n",
```

Comment:
Improved error message - We used to talk about type annotations, but the problem with this test case is the map type, not the type annotation.